analyze.sh

```diff
@@ -1,16 +1,106 @@
 #!/bin/bash
 
+# set com.example.demo and all chid packages (.- means all children, .* this package only)
+# PACKAGES="com.example.demo.-"
+# source path to prepend to the class path
+# BASEPATH="src/main/java"
+# DEPENDENCIES_PATH="~/.m2"
+# OUTPUT_TYPE="sarif"
+
 # Check whether to use latest version of PMD
-if [ "$SPOTBUGS_VERSION" == 'latest' ]; then
+if [ "$SPOTBUGS_VERSION" == 'latest' ] || [ "$SPOTBUGS_VERSION" == "" ]; then
     LATEST_TAG="$(curl -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/spotbugs/spotbugs/releases/latest | jq --raw-output '.tag_name')"
     SPOTBUGS_VERSION=$LATEST_TAG
 fi
 
 # Download SpotBugs
-wget https://github.com/spotbugs/spotbugs/releases/download/"${SPOTBUGS_VERSION}"/spotbugs-"${SPOTBUGS_VERSION}".zip
-unzip spotbugs-"${SPOTBUGS_VERSION}".zip
+wget -q -N https://github.com/spotbugs/spotbugs/releases/download/"${SPOTBUGS_VERSION}"/spotbugs-"${SPOTBUGS_VERSION}".zip
```

Comment:
This change serves to avoid a too long log that is not related to the actual SAST review